src/core/utils/helpers/list-mapper.ts

```diff
@@ -0,0 +1,161 @@
+import { LoanV2, RenewedLoanV2 } from "../../fbs/model";
+import { FaustId } from "../types/ids";
+import { store } from "../../store";
+import { GetMaterialManifestationQuery } from "../../dbc-gateway/generated/graphql";
+import { BasicDetailsType } from "../types/basic-details-type";
+import { Product, Loan } from "../../publizon/model";
+import { LoanType } from "../types/loan-type";
+
+// Creates a "by author, author and author"-string
+// String interpolation todo?
+export const getContributors = (creators: string[] | undefined) => {
```

Comment:
Why can creators be undefined? It is maybe a matter of taste but I think it is better to keep it strictly to `creators: string[]` and make sure in the code that references `getContributors(creators)` check if `creators` is not undefined.

Reply:
Another thing: We have a LOT of places where we are showing a string with authors/contributors. Can this be used in all those places?

Reply:
I don't know. Probably if the input fits, but there are some strings that it is using ("and","or"). But I think maybe this will be changed when the string interpolation stuff is created?